### PR TITLE
Include NovaSeqXPlus in xml genetation

### PR DESCRIPTION
**Pull request by ssjunnebo**

### CODE REVIEW

File: taca_ngi_pipeline/utils/nbis_xml_generator.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def generate_xml_and_manifest(self, return_string_dict=False):
             exml.write(experiment_set)
             rxml.write(run_set)
 
-    def _generate_manifest_file(self, exp_details, run_details):
+    def _generate_manifest_file(self, exp_details):
```
Comment (alneberg) [inline taca_ngi_pipeline/utils/nbis_xml_generator.py:98]: Is this a bugfix?

Comment (alneberg) [reply taca_ngi_pipeline/utils/nbis_xml_generator.py:98]: Ah, no saw it now

**Review by alneberg — APPROVED** [review]
Tipp topp! 👍

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**